### DISCUSSION TITLE
APU/Displays doesn't work

### DISCUSSION THREAD

**Discussion by Martas-007** [flybywiresim/aircraft#3877] [Help]
Hello to all, 

I would like to ask you for help. Unfortunately I didn't find any solution in discussion for problem I have. 

I have installed last version 0.5.3 via installation program, where I choose stable version.
When I want to start ("wake up") the aircraft, I'm not able to do it. I push both battery buttons, then I hear some signal, but both displays, which should show numbers - battery voltage, show nothing. But radio starts normally and I'm able to order power supply by ground service, cockpit illumination works as well. 
I push External power (tried to do it with and without power supply provided by ground service) and I hear , that the fans start work, I set rotary knobs PFD and ND in BRT position, but nothing happens and all displays are still dark.  I push the APU master and then APU start, but the APU never goes into the green AVAIL mode.
Here in discussion I found some solving of problem with APU by the deleting of UWA lighting mode, but I have not installed any other mode and didn't find it in my computer. 
I found some recommendation in another forum and I have set the flight mode to "modern" no legacy , and assistance to "True to life" (except of Taxi way)
Also I tried to reinstall again this whole mode.

FS2020 I have 2 weeks, so I don't have possibility to compare functionality with previous versions of this mode.
In default mode of A320 by Microsoft all the systems works without problems.

So do you have any idea what I do wrong?

thanks a lot

Martin

**Comment by Martas-007** [comment]
Oh now I see Riveroalbert had the same problem and write question before me and also the solving through some update od MFS. So i try tomorrow to do the same... but not sure if the problem will solve also the APU problem. I will try.

**Comment by riveroalbert** [comment]
Hi, try updating now. It is fixed for me. Was some mess up from MFS developers I guess

**Comment by Martas-007** [comment]
Hi, I have updated MFS, but the problem was not solved. The displays are still black and APU doesn't start. After update I tried to uninstall this mode (delete the folder A32NX), and start with default A320 by Microsoft and all worked fine. Then I installed this mode again (via FlyByWire Installer) and I'm back in problem :-(
![1](https://user-images.githubusercontent.com/80358497/110599483-3122df80-8183-11eb-95e0-5c1bed00445d.jpg)
![2](https://user-images.githubusercontent.com/80358497/110599484-32540c80-8183-11eb-95a6-abeb7950e27b.jpg)

**Comment by Martas-007** [comment]
It looks develpoment version works, stable not :-)
